docs/extensions/basics/web-assets.md

```diff
@@ -110,8 +110,8 @@ In the above example:
   * `setFileName()` sets the file name
   * `setLate(true)` tells the asset injector to insert the `<link rel="stylesheet" href="path/web/koala.css">`
     at the end of the HTML `<body>`
-  * `setPriority(99)` tells the injector to insert this `<link rel="stylesheet" href="path/web/koala.css">`
-    *after* any other stylesheet tags in that location with a priority lower than 99
+  * `setPriority(5)` tells the injector when to insert the stylesheet. Lower
```

Comment:
Lower first is the way it's handled for css, js and widgets currently (I tested all three, @bobdenotter tested widgets and verified it), so these docs seem correct for now at least. Changing it to the other way around makes sense to me, but doing it now might break extensions, no?